llvm/lib/ProfileData/Coverage/CoverageMapping.cpp

```diff
@@ -566,14 +567,14 @@ static unsigned getMaxBitmapSize(const CounterMappingContext &Ctx,
                                  const CoverageMappingRecord &Record) {
   unsigned MaxBitmapID = 0;
   unsigned NumConditions = 0;
-  // The last DecisionRegion has the highest bitmap byte index used in the
-  // function, which when combined with its number of conditions, yields the
-  // full bitmap size.
+  // Scan max(BitmapIdx).
+  // Note that `<=` is used insted of `<`, because `BitmapIdx == 0` is valid
+  // and `MaxbitmapID is `unsigned`. `BitmapIdx` is unique in the record.
   for (const auto &Region : reverse(Record.MappingRegions)) {
```

Comment:
Since you need to check all MCDCDecisionRegions, don't need to look in reverse anymore.

Reply:
I left it as is because larger BitmapIdx can be met earlier. :)

Reply:
Yes, that's true -- then you wouldn't need to be constantly updating the variable on every iteration.  Disregard my comment.